src/Pulp/Init.purs

```diff
@@ -107,22 +137,51 @@ init initStyle force out = do
     when (dir /= cwd) (mkdirIfNotExist dir)
     writeTextFile UTF8 f.path f.content
 
-  install initStyle
+  install initStyle effOrEffect
 
   where
-    install Bower = do
+    install Bower UseEff = do
       launchBower ["install", "--save", "purescript-prelude", "purescript-console"]
       launchBower ["install", "--save-dev", "purescript-psci-support"]
 
-    install PscPackage = do
+    install Bower UseEffect = do
+      launchBower ["install", "--save", "purescript-prelude", "purescript-console", "purescript-effect"]
+      launchBower ["install", "--save-dev", "purescript-psci-support"]
+
+    install PscPackage UseEff = do
       launchPscPackage ["init"]
       launchPscPackage ["install", "eff"]
       launchPscPackage ["install", "console"]
       launchPscPackage ["install", "psci-support"]
 
+    install PscPackage UseEffect = do
+      launchPscPackage ["init"]
+      launchPscPackage ["install", "effect"]
+      launchPscPackage ["install", "console"]
+      launchPscPackage ["install", "psci-support"]
+
 action :: Action
 action = Action \args -> do
-  force      <- getFlag "force" args.commandOpts
-  pscPackage <- getFlag "pscPackage" args.globalOpts
-  out        <- getOutputter args
-  init (if pscPackage then PscPackage else Bower) force out
+  force       <- getFlag "force" args.commandOpts
+  pscPackage  <- getFlag "pscPackage" args.globalOpts
+  withEff     <- getFlag "withEff" args.commandOpts
+  withEffect  <- getFlag "withEffect" args.commandOpts
+  out         <- getOutputter args
+  effOrEffect <- getEffOrEffect out withEff withEffect
+
+  if withEff && withEffect
+    then throw $ "Cannot specify both --with-eff and --with-effect. Please choose one and try again."
+    else init (if pscPackage then PscPackage else Bower) effOrEffect force out
+
+  where
+
+  minEffectVersion = HVer.Version (fromFoldable [0, 12, 0]) Nil
+
+  getEffOrEffect out withEff withEffect = do
```

Comment:
Is there a nicer way to write this function? These if/elses make my skin itch a bit.